.travis.yml

```diff
@@ -3,7 +3,8 @@ language: node_js
 stages:
   - lint and build
   - test
-  - release
+  - name: release
+    if: branch = master
```

Comment:
Don't know if this is wanted but I could not explain why it was unable to find `@semantic-release/git` module